govuk-one-login/onboarding-self-service-experience
# Ato 1237/update consumer tests

# Onboarding Feature Deployment

Consumer test for POST and PUT request for client registry

Updated test to call handler instead of mocking out.
Added type for response object. (Maybe assert it in handler later)
Remove initial setup (verification)

## How to review
 - make sure no changes to production code
 - check we are asserting the right checks for the data in the pact contract
